File: app/src/main/java/ml/sky233/suiteki/util/MsgUtils.java
package ml.sky233.suiteki.util;

import android.os.Message;

public class MsgUtils {

    public static Message build(Object obj,int what){
        Message msg = Message.obtain();
        msg.obj = obj;
        msg.what = what;
        return msg;
    }

    public static Message build(int what){
        Message msg = Message.obtain();
        msg.obj = "";
        msg.what = what;
        return msg;
    }

}

File: app/src/main/java/ml/sky233/suiteki/util/MsgUtils.kt
package ml.sky233.suiteki.util

import android.os.Message

object MsgUtils {
    fun build(obj: Any?, what: Int): Message {
        val msg = Message.obtain()
        msg.obj = obj
        msg.what = what
        return msg
    }

    fun build(what: Int): Message {
        val msg = Message.obtain()
        msg.obj = ""
        msg.what = what
        return msg
    }
}